src/ProductConstructionService/ProductConstructionService.Common/MetricRecorder.cs

```diff
@@ -16,7 +16,7 @@ public class MetricRecorder : IMetricRecorder
     public const string PcsMetricsNamespace = "ProductConstructionService.Metrics";
     private const string WaitTimeMetricName = "pcs.queue.wait_time";
 
-    private readonly Counter<int> _queueWaitTimeCounter;
+    private Counter<int> _queueWaitTimeCounter;
```

Comment:
Why this change?

Reply:
`Interlocked` works with `ref` and `readonly` fields cannot be used as `ref`s

Reply:
Oh, my bad, I change how it works; will change it back